src/holochainClient.js

```diff
@@ -101,23 +105,33 @@ export function createZomeCall (zomeCallPath, callOpts = {}) {
       }
       return result
     } catch (error) {
-      console.log(
-        `👎 %c${zomeCallPath}%c zome call ERROR using args: `,
-        'font-weight: bold; color: rgb(220, 208, 120); color: red',
-        'font-weight: normal; color: rgb(160, 160, 160)',
-        args,
-        ' -- ',
-        error
-      )
+      const repeatingError = prevErr.find(e => e.path === zomeCallPath && e.error === error)
```

Comment:
what's the motivation for this repeating error catching?

Reply:
To have more clearer and more controlled log output during integration testing.